README.md

```diff
@@ -184,826 +35,59 @@ $ ./bloaty bloaty -d compileunits
-  This is the default.
-* `-C full` or `-d fullsymbols`: full demangling.
+For more information about the analysis performed by Bloaty,
+please see [How Bloaty Works](doc/how-bloaty-works.md).
 
-One very handy thing about `-C short` (the default) is that
-it groups all template instantiations together, regardless
-of their parameters.  You can use this to determine how much
-code size you are paying by doing multiple instantiations of
-templates.  Try `bloaty -d shortsymbols,fullsymbols`.
 
-## Input Files
+## Install
 
-When you pass multiple files to Bloaty, the `inputfiles`
-source will let you break it down by input file:
+To build, use `cmake`. For example:
 
-```cmdoutput
-$ ./bloaty -d inputfiles CMakeFiles/libbloaty.dir/src/*.o
-    FILE SIZE        VM SIZE    
- --------------  -------------- 
-  42.8%  1.09Mi  37.9%   116Ki    CMakeFiles/libbloaty.dir/src/bloaty.cc.o
-  15.7%   407Ki  15.5%  47.6Ki    CMakeFiles/libbloaty.dir/src/dwarf.cc.o
-  10.3%   266Ki  10.4%  31.8Ki    CMakeFiles/libbloaty.dir/src/bloaty.pb.cc.o
-   9.0%   232Ki   9.7%  29.8Ki    CMakeFiles/libbloaty.dir/src/elf.cc.o
-   8.0%   207Ki   8.7%  26.6Ki    CMakeFiles/libbloaty.dir/src/macho.cc.o
-   4.4%   114Ki   4.3%  13.1Ki    CMakeFiles/libbloaty.dir/src/webassembly.cc.o
-   4.0%   103Ki   7.5%  22.9Ki    CMakeFiles/libbloaty.dir/src/demangle.cc.o
-   3.4%  87.0Ki   3.3%  10.2Ki    CMakeFiles/libbloaty.dir/src/range_map.cc.o
-   2.5%  64.3Ki   2.6%  7.94Ki    CMakeFiles/libbloaty.dir/src/disassemble.cc.o
- 100.0%  2.53Mi 100.0%   306Ki    TOTAL
 ```
-
-## Archive Members
-
-When you are running Bloaty on a `.a` file, the `armembers`
-source will let you break it down by `.o` file inside the
-archive.
-
-```cmdoutput
-$ ./bloaty -d armembers liblibbloaty.a
-    FILE SIZE        VM SIZE    
- --------------  -------------- 
-  28.5%  1.21Mi  23.5%   120Ki    cxa_demangle.cpp.o
-  25.6%  1.09Mi  22.6%   116Ki    bloaty.cc.o
-   9.4%   407Ki   9.3%  47.6Ki    dwarf.cc.o
-   6.2%   266Ki   6.2%  31.8Ki    bloaty.pb.cc.o
-   5.4%   232Ki   5.8%  29.8Ki    elf.cc.o
-   4.8%   207Ki   5.2%  26.6Ki    macho.cc.o
-   2.6%   114Ki   2.6%  13.1Ki    webassembly.cc.o
-   2.4%   103Ki   4.5%  22.9Ki    demangle.cc.o
-   2.0%  87.0Ki   2.0%  10.2Ki    range_map.cc.o
-   1.9%  80.4Ki   3.2%  16.7Ki    charconv_bigint.cc.o
-   1.8%  79.3Ki   2.7%  14.0Ki    escaping.cc.o
-   1.5%  65.0Ki   2.1%  10.9Ki    [9 Others]
-   1.5%  64.3Ki   1.5%  7.94Ki    disassemble.cc.o
-   1.4%  59.9Ki   0.0%       0    [AR Symbol Table]
-   1.0%  45.2Ki   2.4%  12.4Ki    numbers.cc.o
-   0.9%  40.9Ki   2.2%  11.4Ki    charconv.cc.o
-   0.9%  38.8Ki   1.2%  6.10Ki    int128.cc.o
-   0.7%  30.1Ki   1.1%  5.58Ki    str_cat.cc.o
-   0.6%  24.1Ki   0.8%  3.92Ki    string_view.cc.o
-   0.5%  21.2Ki   0.6%  3.21Ki    throw_delegate.cc.o
-   0.4%  19.2Ki   0.4%  2.26Ki    ascii.cc.o
- 100.0%  4.23Mi 100.0%   512Ki    TOTAL
+$ cmake -G Ninja .
+$ ninja
```

Comment:
```suggestion
$ cmake -B build -G Ninja -S .
$ ninja -C build
$ ninja -C build -t install
```

This ensures that the copy-paste will run out-of-tree builds, and uses the `install` target to generate a generally unix-like install image.

Reply:
`ninja -C build -t install` fails on my machine:

```
ninja: fatal: unknown tool 'install'
```

Also should we use a cmake invocation for the second two commands so it will work even if the user doesn't decide to use `ninja`?

Reply:
Updated the suggestion to use `cmake` to drive the build tool, that should make it easier to follow/update and fix the error.